docs/conf.py

```diff
@@ -109,7 +109,7 @@
         'For the stable version documentation, see <a href="/en/stable/">here</a>.'
     ),
     '2.x-line': (
-        'marshmallow 2 will no longer be supported as of 2020-08-18. '
+        'marshmallow 2 is no longer be supported as of 2020-08-18. '
         '<a href="https://marshmallow.readthedocs.io/en/latest/upgrading.html#upgrading-to-3-0">'
```

Comment:
s/is no longer be supported/is no longer supported/

Reply:
Doh, will fix